README.md

```diff
@@ -67,10 +67,45 @@ winpty docker exec -it vncko bash
 ```
 From here, you can edit the frontend code. When you want to see your changes reflected, you can **Restart KOReader** from the emulator KOReader system menu.
 
+### Updated KOReader
+Allows you to work with a more recent version of KOReader.
```

Comment:
I'm not really sure what this means. Shouldn't you simply update the one above if something about it is outdated?

Reply:
I thought there might be a reason to work with a 2020 version of KOReader, but maybe I'm wrong. It's not my repo so I'd rather add than replace. It's up to you. I'm just sharing my meager experience of what I ended up with today after following your instructions.
In any case, with the current code, if you only modify the KOReader version for a newer one, without changing the rest of the code, you end up with dependency problems and linux versions that are too old. That's why I used Debian Bullseye and installed these dependencies.

Reply:
Ohh, now I get it, well, the problem anyway. The `latest` tag hasn't been updated in a while because it's not done automatically anymore. Maybe it shouldn't even be there at all. The correct current image is `koreader/koappimage:0.4.4-20.04`.

But all old versions still work on the latest systems.